setup.py

```diff
@@ -108,8 +104,8 @@ def download_SNI() -> None:
             # prefer "many" builds
             if "many" in download_url:
                 break
-            # prefer the correct windows or windows7 build
-            if platform_name == "windows" and ("windows7" in download_url) == (sys.version_info < (3, 9)):
+            # prefer the correct windows build
```

Comment:
I think the comment can be a lot clearer
```suggestion
            # prefer non-windows7 builds to get up-to-date dependencies
```